geomesa-wps/src/main/scala/org/locationtech/geomesa/process/ProcessFactory.scala

```diff
@@ -15,7 +15,7 @@
  */
 
 
-package org.locationtech.geomesa.plugin.process
+package org.locationtech.geomesa.process
```

Comment:
should we try to make the module name and package name consistent? I think all our other modules are like that.